test/powershell/Language/Scripting/HashtableToPSCustomObjectConversion.Tests.ps1

```diff
@@ -129,5 +133,20 @@
 	    $obj.PSTypeNames.Count | Should Be 3
 	    $obj.PSTypeNames[0] | Should Be 'System.Object'
     }
+    It "new-object should fail to create object for System.Management.Automation.PSCustomObject" {

```

Comment:
> fail to create object

Any object? `-Property $null` seems like a special case.

Again the same question: is it the right test or it's just the current behavior.